frame/staking-rewards/src/types.rs

```diff
@@ -1,2 +0,0 @@
-//! Internal types used inside pallet, example for storage.
```

Comment:
please move private types of pallet here

Reply:
As I mentioned in other places, let's not over refactor for now, which puts a lot constraints on getting stuff done. Once we have the overall thing working we can do whatever makes sense.